pyccel/ast/core.py

```diff
@@ -2345,7 +2365,8 @@ def __init__(
         cls_parameters=None,
         order='C',
         precision=0,
-        is_argument=False
+        is_argument=False,
+        is_kwonly=False
         ):
```

Comment:
Could you update the docstring of this class too?

And I have a similar comment as above about `__new__`:
```python
def __new__(cls, *args, **kwargs):
    return Basic.__new__(cls)
```